src/main/java/de/dennisguse/opentracks/stats/TrackStatisticsUpdater.java

```diff
@@ -64,6 +65,8 @@
 
     private final AltitudeRingBuffer altitudeBuffer;
     private final SpeedRingBuffer speedBuffer;
+    private float averageHeartRateBPM = 0.0f;
```

Comment:
Can be replaced by calling `resetAverageHeartRate()` during constructor?

Reply:
Done